modular_skyrat/modules/automapper/code/turf_proc.dm

```diff
@@ -0,0 +1,17 @@
+/**
+ * Clear everything
+ *
+ * A more useful version of empty.
+ */
+/turf/proc/clear_everything(turf_type = /turf/open/space, baseturf_type, list/ignore_typecache, flags, list/ignored_atoms = list(/mob/dead))
```

Comment:
Are we actually allowed to use parameters that have default values before parameters that don't have default values? I feel like doing so will force whoever uses this proc to type like `clear_everything(baseturf_type = /what/ever, ignore_typecache = list(/turf/my_turf))` instead of just typing in the values without specifying the arguments.

Reply:
this isn't actually used